autosklearn/automl.py

```diff
@@ -968,8 +972,12 @@ def fit(
             self._logger.info("Finished loading models...")
 
         self._fit_cleanup()
+        self.fitted = True
 
         return self
+    
+    def __sklearn_is_fitted__(self)->bool:
```

Comment:
You'll get an issue about the formatting here
```suggestion
    def __sklearn_is_fitted__(self) -> bool:
```